case-study-hydropro-solutions.mdx

```diff
@@ -0,0 +1,76 @@
+
+With Xata, HydroPro was able to:
+
+- **Store and manage vast amounts of relational data:** Including customer records, water meter data, and work orders.
+- **Use advanced search capabilities:** Enabling quick and accurate data retrieval for customer service, ensuring no billing errors or discrepancies during meter installations.
+- **Manage file attachments:** Storing photos taken during meter installations and using Google Cloud Vision to read meter serial numbers, which are automatically linked to customer records. Xata's image transformation capabilities create smaller thumbnails of the images, allowing field teams using mobile devices to quickly view and manage high-resolution photos in real-time, improving on-site efficiency.
+
+## Implementation
+
+The integration of Xata into HydroPro Solutions' tech stack was seamless. With Next.js as the front-end framework, Vercel handling deployments, and Google Cloud Vision managing image recognition, Xata served as the core Postgres relational database, tying everything together.
+
+One of the most critical features for HydroPro was Xata's ability to handle bulk data uploads, such as CSV files with thousands of addresses and meter readings. Xata automatically processed the data, converted it into JSON, and mapped it into the serverless PostgreSQL database. This automation dramatically improved efficiency, especially during larger water meter installation projects.
+
+![HydroPro Solutions Dashboard](/images/case-study-hydropro/hydropro-dashboard.png)
+
+## Results and impact
+
+Since implementing Xata, HydroPro Solutions has seen significant improvements in their operations, particularly in terms of speed and accuracy. The biggest transformation has been in search functionality, which allows internal teams to quickly locate specific addresses and records across thousands of data points.
+
+**Key benefits include:**
+
+- **60% reduction in search time:** Enabling HydroPro’s teams to resolve customer queries faster and ensure accurate data retrieval.
+- **Efficient project management:** Field teams can track project progress in real-time, correlating installation speeds with environmental factors like temperature and rainfall, which impact productivity.
+- **Improved file handling:** With Xata, HydroPro now stores and manages thousands of photos, making it easy to resolve any customer disputes about water meter readings by referencing high-resolution images.
+
+> With Xata, we’ve achieved a 60% reduction in search time. Previously, searching for an address would lead to long load times, sometimes with the system freezing up. Now, it’s lightning-fast—start typing an address, and results filter instantly. It has truly been a game changer.\\
+>
+> _Chris Hiester, Lead Application Engineer at HydroPro Solutions_
```

Comment:
```suggestion
> With Xata, we’ve achieved a 60% reduction in search time. Previously, searching for an address would lead to long load times, sometimes with the system freezing up. Now, it’s lightning-fast—start typing an address, and results filter instantly. It has truly been a game changer.\
>
> _Chris Hiester, Lead Application Engineer at HydroPro Solutions_
```

Same as above
![CleanShot 2024-09-19 at 11 16 48@2x](https://github.com/user-attachments/assets/2b71d4d3-095d-499a-b3f1-2cfdaba0da5e)